pki.spec

```diff
@@ -845,9 +858,15 @@ fi
     -DWITH_JAVADOC:BOOL=%{?with_javadoc:ON}%{!?with_javadoc:OFF} \
     -DBUILD_PKI_CONSOLE:BOOL=%{?with_console:ON}%{!?with_console:OFF} \
     -DTHEME=%{?with_theme:%{vendor_id}} \
+%if 0%{?rhel}
+    ..
```

Comment:
what does this do?

Reply:
See: https://bugzilla.redhat.com/show_bug.cgi?id=1863276 

Fedora changed things so `%cmake` automatically adds `-B <dir>` flag as below. That's equivalent of `mkdir <dir> && cd <dir> && cmake ..`. RHEL doesn't have that slick option so we have to do that all manually. 

See also: https://pagure.io/fesco/issue/2411 and https://fedoraproject.org/wiki/Changes/CMake_to_do_out-of-source_builds